### PR TITLE
Feat/redesign detail

### PR DESCRIPTION
**Pull request by rayfajars**
## OVERVIEW

1. integration file list request upload
2. add download file (i make this function without test, because server down from 11.00 AM )
3. add filter in table

title: upload file, download file and select filter
project: Digiteam Inventaris Platform
participants: @rayfajars @yoslie @Ibwedagama

### CODE REVIEW

**Review by yoslie — APPROVED** [review]

File: src/components/RequestComponents/FormRequest.vue [hunk 1]
```diff
@@ -108,9 +108,9 @@ export default {
           }
         });
     },
-    onFileChange() {
+    onFileChange(type_file) {
       if (this.$refs.file.files[0]) {
-        const response = sendFile(this.$refs.file.files[0]);
+        const response = sendFile(this.$refs.file.files[0], type_file);
```
Comment (Ibwedagama) [inline src/components/RequestComponents/FormRequest.vue:113]: minor:

hello @rayfajars based on our JS Naming Convention, please use camel case for function argument or variable declaration 👍🏼 

`type_file` -> `typeFile`

ref: https://wiki.digitalservice.id/doc/js-naming-conventions-gRnYoPmhnd

File: src/components/RequestComponents/FormVerifikasiRequest.vue [hunk 2]
```diff
@@ -109,44 +110,23 @@ export default {
...
-      }
-    },
-    setFile(value) {
-      // TODO:  this code i was remove next, if API from backend done, because i want to use code in file inputFile.js
-      const formData = new FormData();
-      formData.append("file", value);
-      this.fileImage = formData;
-    },
-    onFileChangePengembalian(type) {
+    onFileChange(type, format_type_file) {
```
Comment (Ibwedagama) [inline src/components/RequestComponents/FormVerifikasiRequest.vue:113]: minor:

same as the comment above

**Review by Ibwedagama — APPROVED** [review]

File: src/components/RequestComponents/FormRequest.vue (hunk 1 above)
Comment (rayfajars) [reply src/components/RequestComponents/FormRequest.vue]: i am change [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/25/commits/63e11ad17965733270b2deb2cb7b10c8f43060cb)

File: src/components/RequestComponents/FormVerifikasiRequest.vue (hunk 2 above)
Comment (rayfajars) [reply src/components/RequestComponents/FormVerifikasiRequest.vue]: i am change [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/25/commits/63e11ad17965733270b2deb2cb7b10c8f43060cb)

**Pull request merged**